Pendulum/Data/Event+Extensions.swift

```diff
@@ -190,6 +207,15 @@ extension Event {
     
 }
 
+extension Event {
+    static func deleteAll(in context: NSManagedObjectContext) {
+        for event in fetch(from: context) {
```

Comment:
You might want to use [NSBatchDeleteRequest](https://developer.apple.com/documentation/coredata/nsbatchdeleterequest) here?